README.md

```diff
@@ -1,41 +1,39 @@
-- flexibility to run jobs on the bare-metal or in the cloud (support for AWS, plz follows a cloud-agnostic architecture)
 - simple command line interface
-- runs experiments on on-demand or spot instances, using isolated Docker containers
-- tracks parameters for experiments, and allows exact repetitions with tweaked parameters, making experimentation more systematic
-- provides flexibility to use various instances and resource types
-- AMI: use baked images for consistency, repeatability and performance
-- ready to run pytorch in an environment supporting Nvidia GPUs, CUDA and anaconda
-- built on top of broadly used tools like python, bash, make, terraform, ansible and docker
-- useful examples provided see [Examples](#Examples) section
-- MIT-license allowing modification, distribution, private or commercial use, see [License](LICENSE) for more details
+- cloud-agnostic architecture (on top of Docker), allowing you to run jobs locally, on bare metal, or on the cloud
+  - Plz currently supports Amazon Web Services (AWS), but will most likely support other cloud providers in the future
+  - full control of the type of cloud instance, allowing you to use whatever machine fits your job (and budget)
+  - full support for NVIDIA GPUs, allowing you to run deep learning experiments
+- common tooling support, with the following straight out of the box:
+  - Python
+  - Anaconda
+  - PyTorch
+- data-based workflow, so that you can ensure you're always working with the correct inputs and outputs
+- parameter awareness, so that you can run the same experiment with multiple sets of parameters
+- full history, so that you can review your experiments over time
+- useful examples provided (see the [Examples](#Examples) section)
+- MIT-license allowing modification, distribution, private or commercial use (see [LICENSE](LICENSE) for more details)
 - open for contributions, plz
 
 ## Usage overview
 
-We offer more details below on how to setup `plz` and run your jobs, but we can
-start by giving you an overview of what `plz` does.
+We offer more details below on how to setup Plz and run your jobs, but we can start by giving you an overview of what Plz does.
 
-`plz` offers a command-line interface. You start by adding a `plz.config.json`
-file to the directory where you have your source code. This file contains,
-among other things, the command you run to put your program to work (for
-instance, `python main.py`). Then, you can run commands like `plz run`,
-as shown for this example (that is provided in this repository as well):
+Plz offers a command-line interface. You start by adding a `plz.config.json` file to the directory where you have your source code. This file contains, among other things, the command you run to put your program to work (for instance, `python3 main.py`). Then you can run the program with `plz run`. This runs the program in the Plz environment. The following example (provided in this repository) demonstrates this:
```

Comment:
I liked the 80 cols wrapping. Now reading the diff of (not so) long paragraphs is a pain :(

Reply:
It was highly inconsistent—I just picked one. Happy to go with the other one.

Reply:
`prettier --write README.md` will now format the README appropriately. Feel free to set up your editor to do this automatically.

Reply:
OK